analysis_options.yaml

```diff
@@ -1,55 +1 @@
-#   https://www.dartlang.org/guides/language/analysis-options
 include: package:pedantic/analysis_options.yaml
-
-# http://dart-lang.github.io/linter/lints/options/options.html
-linter:
-  rules:
```

Comment:
why remove these? these are applied on top of pedantic rules (mix-and-match).

Reply:
I can revert if I want, but I didn't remove the rules, just instead of declaring exactly which rules I want or not, I decided to use all those recommended by Pedantic.

Reply:
It is the superset of rules that will be applied here, and I'd rather have these rules + pedantic.

Reply:
Done! :)